blobit-core/src/test/java/org/blobit/core/cluster/LedgerLifeTest.java

```diff
@@ -156,9 +158,16 @@ public void testNoTablespaces() throws Exception {
                 // force close all ledgers
                 manager.getBlobManager().closeAllActiveWritersForTests();
 
+                Thread.sleep(10000);
```

Comment:
why are you making this test so slow ?
also Thread.sleep means "flaky tests"

Reply:
I put sleep because when manager.getBlobManager().CloseAllActiveWritersForTests() is called, the thread that should close writers starts late, so we start gc before the writers are closed. But the gc cannot be done if there are active writers. So the test fails.

We can try to handle this problem with a waitForCondition.
What do you think about it?

Reply:
We have this problem only on github, in local even without thread.sleep the tests pass